test/plugins/mergeStyles.02.svg

```diff
@@ -0,0 +1,22 @@
+Check whether plugin works with only one style element (no further merging needed, noop) and a media query.
+
+===
+
+<svg id="test" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
+    <style>.st0{ fill:red; padding-top: 1em; padding-right: 1em; padding-bottom: 1em; padding-left: 1em; }</style>
+    <style>
+        @media screen and (max-width: 200px) { .st0 { display: none; } }
+    </style>
+    <rect width="100" height="100" class="st0" style="stroke-width:3;margin-top:1em;margin-right:1em;margin-bottom:1em;margin-left:1em"/>
+</svg>
+
+@@@
+
+<svg id="test" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
+    <style>
+        .st0{ fill:red; padding-top: 1em; padding-right: 1em; padding-bottom: 1em; padding-left: 1em; }
+
+@media screen and (max-width: 200px) { .st0 { display: none; } }
```

Comment:
Formatting is not currently considered?

Reply:
Compact style output is used. The newlines are added tor better understanding the different merged parts.
The minifyStyles plugin in a step afterwards should then minify the styles as much as possible.

Reply:
Alright, no newlines are added when the collected styles are merged. This reduces the file size a tiny bit. Formatting is not considered by this particular plugin because it only merges styles. But the minifyStyles plugin that uses CSSO could also be used for beautifying styles. But this wouldn't probably be considered an optimization.